### PR TITLE
📝 PPSSPP guide

### LINKED ISSUE #5: 🎮 SNES guide (Snes9x)

### LINKED ISSUE #14: 🎮 PSP guide (PPSSPP)

### PR DESCRIPTION
**Pull request by Vaarlion**
### Proposed changes

Create a small guide for PPSSPP Emulator, should close #14

### CODE REVIEW

File: emulators/ppsspp.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+Go back and select *Exit*, Launch it one last time and in the **Games** tabs on main menu, navigate to wherever your PSP ROMs are - if you used the recommended path, this should be `~/roms/psp`.
+
+## Steam ROM Manager
+
+Open Steam ROM Manager, press Parsers, then enter the following settings:
+
+-   Select `Sony PlayStation Portable - PPSSPP (Flatpak)` under Community Presets.
+-   You can add additional categories to Steam category using the format `${category name}`, this is case-sensitive.
+-   Set executable to `/usr/bin/flatpak`.
+-   Set command line arguments to `run org.ppsspp.PPSSPP "${filePath}"`
```
Comment (nchristopher) [inline emulators/ppsspp.md:40]: Unless I'm missing something, I don't think this is needed; the PPSSPP Flatpak preset already has this filled in.

File: emulators/ppsspp.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Open up the emulator and press **Settings**.
+
+In **Graphics**:
+ * select **Vulkan** as a Backend and say *Yes* to restart now, then launch PPSSPP again.
+ * Go back in Settings, Graphics, then set **AMD RADV VANGOGH** as a Device.
+ * Turn ON **Fullscreen**, **VSync** and set the **Rendering resolution** to **Auto (1:1)**
+ * Set **Upscale level** to **2x** and **Upscale type** to **Hybrid + Bicubic**, then check **Deposterize**
+
+Go back and select *Exit*, Launch it one last time and in the **Games** tabs on main menu, navigate to wherever your PSP ROMs are - if you used the recommended path, this should be `~/roms/psp`.
```
Comment (nchristopher) [inline emulators/ppsspp.md:31]: Minor grammar fix
```suggestion
Go back and select *Exit*. Launch the emulator one last time and in the **Games** tabs on the main menu, navigate to wherever your PSP ROMs are - if you used the recommended path, this should be `~/roms/psp`.
```

File: emulators/ppsspp.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+# PPSSPP Guide
+
+❗ Follow the [Getting Started section](../README.md#getting-started) of `README.md` first.
+
+This section of the guide explains how to setup [PPSSPP](https://www.ppsspp.org), a PSP emulator.
+
+![image](https://user-images.githubusercontent.com/59558433/163827712-fe70ad3e-b625-4439-96ea-40038c128a00.png)
```
Comment (nchristopher) [inline emulators/ppsspp.md:7]: This should have alt text for accessibility reasons.
```suggestion
![A screenshot of PPSSPP's main menu](https://user-images.githubusercontent.com/59558433/163827712-fe70ad3e-b625-4439-96ea-40038c128a00.png)
```

**Review by nchristopher — CHANGES_REQUESTED** [review]
Thanks for the guide; it looks really good! Just a few small notes.

File: emulators/ppsspp.md (hunk 1 above)
Comment (Vaarlion) [reply emulators/ppsspp.md:40]: 100% but so does the dolphin flatpak and it was specified, i though it was on purpose do i did the same :)
I can remove it

Comment (nchristopher) [reply emulators/ppsspp.md:40]: The Dolphin guide doesn't use the Flatpak yet.

Comment (Vaarlion) [reply emulators/ppsspp.md:40]: And that show how good i am at reading :flushed: 
thank's !

**Comment by Vaarlion** [comment]
Should i squash ?

**Comment by nchristopher** [comment]
No, when I merge I'll squash. I still need to test this out myself; other than that, it should be ready to merge.

**Review by nchristopher — APPROVED** [review]

**Pull request merged**